pytest.ini

```diff
@@ -34,4 +34,4 @@ filterwarnings =
     error::pytest.PytestCollectionWarning
 markers =
     need_serialized_dag
-asyncio-mode = strict
+asyncio_mode = strict
```

Comment:
Interesting...
We get warnings in the CI that this is not right? but it is right :thinking:
```
 /home/runner/work/airflow/airflow/scripts/.build/.docker_venv/host_python_3.7/lib/python3.7/site-packages/_pytest/config/__init__.py:1252: PytestConfigWarning: Unknown config option: asyncio_mode
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")
```
https://github.com/apache/airflow/runs/5772462748?check_suite_focus=true#step:7:547

Reply:
Interesting